### PR TITLE
portfolio-branch.

### PR DESCRIPTION
**Pull request by direakanbi**
# My portfolio
This project was created for the creation of my portfolio.

# LANGUAGES USED
1. HTML
2. CSS
3. Linters(Lighthouse, Webhint, Stylelint
# PREREQUISITES
1. Prior knowledge of HTML and CSS.
2. Setup: clone direakanbi/microverse-portfolio1 into your local machine
3. Run index.html on your local server.

# Live Demo Link.
http://127.0.0.1:5500/portfolio1/index.html#

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
...
+1. HTML
+2. CSS
+3. Linters(Lighthouse, Webhint, Stylelint
+# PREREQUISITES
+1. Prior knowledge of HTML and CSS.
+2. Setup: clone direakanbi/microverse-portfolio1 into your local machine
+3. Run index.html on your local server.
+
+# Live Demo Link.
+http://127.0.0.1:5500/portfolio1/index.html#
```
Comment (AshakaE) [inline README.md]: - [ ] Please update this link because `127.0.0.1` will only work on the host machine

File: index.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;600&display=swap" rel="stylesheet" >
+</head>
+<body>
+  <main>
+      <div class="header">
+       <a href="">My Logo</a>
+      <nav><a href=""><i class="fas fa-bars"></i></a></nav>
+      </div>
```
Comment (AshakaE) [inline index.html]: The container for this section is a `header` tag

- [ ] Kindly use a `header` tag in place of `div`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      <div class="header">
+       <a href="">My Logo</a>
+      <nav><a href=""><i class="fas fa-bars"></i></a></nav>
+      </div> 
+    <section class="container">
+      <div class="intro">
+        <h1>Hi! I'M DIRE</h1> 
+          <h1>GLAD TO SEE <br>YOU!</h1>
+         <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      </div>
```
Comment (AshakaE) [inline index.html]: The font properties used here do not match the template to a certain level, this includes; font face, font weight, and font color

- [ ] Kindly update it accordingly, see reference below

Figma | Your copy 
------------ | ------------ 
image | image

File: README.md (hunk 1 above)
Comment (AshakaE) [inline README.md]: - [ ] I suggest using the Microverse README template to create a more robust README 🙂

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @direakanbi  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


#  Required Changes ♻️ 🚧
_Check the comments under the review._

Your deployment link in your pull request is from your local machine which will not work everywhere except on the machine that hosts it

- [ ] Kindly use GitHub pages or Netlify to deploy your live demo


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mke2111 — APPROVED** [review]
### Hi @direakanbi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**